README.md

```diff
@@ -84,6 +98,18 @@ conda activate web-ui
 - Verify you have access to run the `run_webui_mac.sh` script by running `chmod +x run_webui_mac.sh` in your terminal
 - Run `./run_webui_mac.sh`
 
+## Config
+
+The `ui-config.json` is used to configure all of the settings for the ui including the limits of the components/elements (min/max values). If this file is not present the limits are hard coded to a default value that is in the python code and written to the `ui-config.json` file at the root of the project.
+Configuration files like `ui-config.json` are not included in the repository, and are excluded using the [.gitignore](.gitignore) file. 
+
+Updates to the defaults in the python code should been done instead to add greater limits that we can then temper with the `ui-config.json`.
+
+Hard coded default values have been changed in the following components according to the needs of AIMC:
+
+- [scripts/loopback.py](scripts/loopback.py)  
+  - [loopback slider](https://github.com/AIM-Inc/stable-diffusion-webui/blob/e4e9f83f3754443a21facc2bf8e56a9aba420eaa/scripts/loopback.py#L20) max value is now set to `100000`

```

Comment:
nit: Mostly nit, but I've been thinking about where it makes the most sense to record changes to important/unique values such as these. I don't have a good answer at the moment so just something to think about as we continue getting more requirements similar to this. I'd like to have a place where in the future we can go back and _understand_ why we did this. Hope it makes sense.